packages/docs/src/components/features/ColorPalette.vue

```diff
@@ -66,54 +66,39 @@
   </section>
 </template>
 
-<script>
+<script setup lang="ts">
   // Utilities
   import kebabCase from 'lodash/kebabCase'
-  import colors from 'vuetify/lib/util/colors'
+  import { colors as vuetifyColors } from 'vuetify/lib/util/colors'
+  import { computed } from 'vue'
 
-  export default {
-    name: 'ColorPalette',
+  type Colors = Record<string, Record<string, string>>
```

Comment:
My feeling is that `Colors` is not a type that is exclusive to the `ColorPalette` component, therefore perhaps it should live elsewhere?

Either in `'vuetify/lib/util/colors'` or `colors` should already be typed, what do you think?